### ISSUE TITLE
HttpClientHandler.AutomaticDecompression documentation incorrectly states that the default value is none.

### ISSUE THREAD

**Issue by himadrisarkar** [dotnet/docs#1054]
HttpClientHandler.AutomaticDecompression's [documentation ](https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclienthandler.automaticdecompression?view=netcore-1.1#System_Net_Http_HttpClientHandler_AutomaticDecompression) states that default value is None when in the actual implementation it is DecompressionMethods.GZip | DecompressionMethods.Deflate. Please update the documentation.

**Comment by davidsh** [comment]
We will need the documentation to have multiple sections.  The default for .NET Framework 4.x System.Net.Http binary (i.e. the GAC'd one) is `None`. However, the .NET Core version has a different default: `DecompressionMethods.GZip | DecompressionMethods.Deflate`.

Further adding some confusion is that the 4.1 System.Net.Http NuGet package carries an OOB version of the System.Net.Http implementation (from .NET Core). This library package can be used on .NET Framework (Desktop).  And that means it will have the default of `DecompressionMethods.GZip | DecompressionMethods.Deflate`.

So, only the original .NET Framework Desktop binary has the default of `None`.

**Comment by mairaw** [comment]
@davidsh is there any plan to make this behavior consistent across platforms? I can change the default value for now on docs, but once we have all the .NET docs in one place, we need to able to explain how this works to users.

**Comment by davidsh** [comment]
cc: @stephentoub @weshaggard @ericstj 

>is there any plan to make this behavior consistent across platforms?

At the time we first did .NET Core, we made we made a conscious decision to change the default from .NET Framework to be more useful which is why it is not `None`.  I know this makes the docs difficult especially since System.Net.Http OOB's on Desktop.  Customer feedback has been positive so far on this change so, we aren't thinking that we would roll it back to match .NET Framework Desktop, although I will say that this is an exception since most other things do match behaviors between .NET Core and .NET Framewok.
